runtime/verutil/cfrerr.c

```diff
@@ -78,6 +78,82 @@ getJ9CfrErrorBsmMessage(J9PortLibrary* portLib, J9CfrError* error, const U_8* cl
 	return errorString;
 }
 
+const char*
+getJ9CfrErrorMajorVersionMessage(J9PortLibrary* portLib, J9CfrError* error, const U_8* className, UDATA classNameLength)
+{
+	PORT_ACCESS_FROM_PORT(portLib);
+	UDATA allocSize = 0;
+	char *errorString = NULL;
+
+	const char *template = j9nls_lookup_message(J9NLS_ERROR | J9NLS_DO_NOT_APPEND_NEWLINE, J9NLS_CFR_ERR_MAJOR_VERSION2, NULL);
+
+	allocSize = strlen(template) + classNameLength + (MAX_INT_SIZE * 4) + 1;
+	errorString = j9mem_allocate_memory(allocSize, OMRMEM_CATEGORY_VM);
```

Comment:
Please reorder the declarations so each variable get its value in that declaration:
```
	const char *template = j9nls_lookup_message(J9NLS_ERROR | J9NLS_DO_NOT_APPEND_NEWLINE, J9NLS_CFR_ERR_MAJOR_VERSION2, NULL);
	UDATA allocSize = strlen(template) + classNameLength + (MAX_INT_SIZE * 4) + 1;
	char * errorString = j9mem_allocate_memory(allocSize, OMRMEM_CATEGORY_VM);
```
Also do the same for each of the new methods below.